src/test/java/org/takes/http/BkBasicTest.java

```diff
@@ -86,6 +102,59 @@ public void handlesSocket() throws IOException {
     }
 
     /**
+     * BkBasic can handle HTTP persistent connections.
+     * @throws Exception If some problem inside
+     * @checkstyle ExecutableStatementCountCheck (2 lines)
+     */
+    @Test
+    @SuppressWarnings("PMD.DoNotUseThreads")
+    public void handlesPersistentConnection() throws Exception {
+        final int port = 8080;
+        final String uri = String.format("http://localhost:%d", port);
+        final Take take = new Take() {
+            @Override
+            public Response act(final Request req) throws IOException {
+                return new TkEmpty().act(req);
+            }
+        };
+        final Thread app = new Thread(
+                new Runnable() {
+                    @Override
+                    public void run() {
+                        try {
+                            new FtBasic(
+                                    new BkBasic(take),
+                                    port
+                            ).start(
+                                    Exit.NEVER
+                            );
+                        } catch (final IOException ex) {
+                            throw new IllegalStateException(ex);
+                        }
+                    }
+                }
+        );
+        app.start();
+        Thread.sleep(BkBasicTest.WAIT_APP);
+        final HttpURLConnection persistConnection = HttpURLConnection
```

Comment:
@mstzn36 pelase checkout this article http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#variable-names
>Avoid composite names of variables, like timeOfDay, firstItem, or httpRequest